instagram-app/src/components/SearchBar/SearchBar.js

```diff
@@ -1,29 +1,63 @@
 import React from 'react';
 import './SearchBar.css';
+import styled from 'styled-components';
 
-let SearchBar = props =>{
+
+const Header = styled.header`
+    display:flex;
+    justify-content: space-between;
+    align-items: center;
+    height:100px;
+    max-width: 1900px;
+    margin: 20px  auto;
+`;
+
+const Left = styled.div`
+    display:flex;    
+`
+
+const Right = styled.div`
+    padding:5px;
+`;
+
+const Logopic = styled.img`
+    height: 80px;
+`;
+
+
+const Search = styled.input`
+    text-align: center;
+    padding: 10px 50px;
+    font-size: 30px;
+    border:1px solid lightgray;
+    border-radius: 4px;
+`;
+
+const Center = styled.div`
+`;
+
+
+let SearchBar = props => {
     return (
-        <div className = 'header'>
-            <div className = 'left'>
+        <Header>
+            <Left>
                 <i class="fab fa-instagram fa-4x"></i>
```

Comment:
In React, the `className` attribute should be used instead of `class` (line 44).